### DISCUSSION TITLE
More than clock, taskbar icons too!

### DISCUSSION THREAD

**Discussion by iwillforce** [martinet101/ElevenClock#437] [General]
Hi there,
Due to some professional and personal reasons, showing clock in secondary montir is super! But, im missing so much the option to show taskbar icons too.
There's any way/possible to add this feature?

Best regards,
Will

**Comment by marticliment** [comment]
Im sorry, but this is not possible. There's an issue or dicussion somewhere explaining the reasons, but basically they are win32api limitations
